public/app/features/profile/UserProfileEditPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useAsync } from 'react-use';
+import { connect, ConnectedProps } from 'react-redux';
+import { hot } from 'react-hot-loader';
+import { NavModel } from '@grafana/data';
+import { VerticalGroup } from '@grafana/ui';
+
+import { getNavModel } from 'app/core/selectors/navModel';
+import { StoreState } from 'app/types';
+import Page from 'app/core/components/Page/Page';
+import { changeUserOrg, initUserProfilePage, revokeUserSession, updateUserProfile } from './state/actions';
+import UserProfileEditForm from './UserProfileEditForm';
+import SharedPreferences from 'app/core/components/SharedPreferences/SharedPreferences';
+import { UserTeams } from './UserTeams';
+import UserOrganizations from './UserOrganizations';
+import UserSessions from './UserSessions';
+
+export interface OwnProps {
+  navModel: NavModel;
+}
+
+function mapStateToProps(state: StoreState) {
+  const userState = state.user;
+  const { user, teams, orgs, sessions, teamsAreLoading, orgsAreLoading, sessionsAreLoading, isUpdating } = userState;
+  return {
+    navModel: getNavModel(state.navIndex, 'profile-settings'),
+    orgsAreLoading,
+    sessionsAreLoading,
+    teamsAreLoading,
+    orgs,
+    sessions,
+    teams,
+    isUpdating,
+    user,
+  };
+}
+
+const mapDispatchToProps = {
+  initUserProfilePage,
+  revokeUserSession,
+  changeUserOrg,
+  updateUserProfile,
+};
+
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+export type Props = OwnProps & ConnectedProps<typeof connector>;
+
+export function UserProfileEditPage({
+  navModel,
+  orgsAreLoading,
+  sessionsAreLoading,
+  teamsAreLoading,
+  initUserProfilePage,
+  orgs,
+  sessions,
+  teams,
+  isUpdating,
+  user,
+  revokeUserSession,
+  changeUserOrg,
+  updateUserProfile,
+}: Props) {
+  useAsync(async () => {
+    await initUserProfilePage();
+  }, []);
+
+  return (
+    <Page navModel={navModel}>
+      <Page.Contents isLoading={!Boolean(user)}>
```

Comment:
```suggestion
      <Page.Contents isLoading={!user}>
```
`Boolean` is redundant here since `!` will convert the value to boolean.

Reply:
Good catch!

Reply:
Fixed!